prow/integ-suite-kind.sh

```diff
@@ -39,21 +39,18 @@ export TEST_ENV=kind
 # See https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster
 export PULL_POLICY=IfNotPresent
 
-export HUB=${HUB:-"istio-testing"}
-export TAG="${TAG:-"istio-testing"}"
+export HUB=${HUB:-"gcr.io/istio-testing"}
+export TAG="${TAG:-"latest"}"
 
 # Setup junit report and verbose logging
 export T="${T:-"-v"}"
 export CI="true"
 
-make init
-
 if [[ -z "${SKIP_SETUP:-}" ]]; then
   time setup_kind_cluster "${NODE_IMAGE:-}"
 fi
 
 if [[ -z "${SKIP_BUILD:-}" ]]; then
-  time build_images
   time kind_load_images ""
```

Comment:
Also not needed, this is only needed for local images which are not pushed. For this case, we can skip it and kubernetes will pull the images like normal